ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -767,6 +767,34 @@ def test_trilu(*, dtype_and_x, k, upper, test_flags, backend_fw, fn_name, on_dev
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.experimental.unsorted_segment_mean",
+    d_x_n_s=valid_unsorted_segment_min_inputs(),
+    test_with_out=st.just(False),
+    test_gradients=st.just(False),
+)
+def test_unsorted_segment_mean(
+    *,
+    d_x_n_s,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+):
+    dtypes, data, num_segments, segment_ids = d_x_n_s
+    if backend_fw == "numpy,torch,jax,paddle":
```

Comment:
not sure why this is being hard-coded?

Reply:
Hi @rishabgit ,

We can remove this and still test cases are passed successfully. I checked in my local.

Reply:
This condition is set to restrict the tests to exclude backends that are not currently compatible with the functions being tested.